packages/core-data/README.md

```diff
@@ -40,6 +40,60 @@ function MyAuthorsListBase() {
 }
 ```
 
+## What's an entity?
+
+An entity represents a WordPress REST API endpoint. Each item within the entity is called entity record. Available entities are defined in `rootEntitiesConfig` at ./src/entities.js.
```

Comment:
For me REST API is just an implementation detail and my ultimate Goal has always been to decouple from the WordPress REST API using an adapter layer, so in that sense, I'm not a big fan for tieing a definition of an entity to the backend.

I'd define an entity as a type of "model/data" or something like that that the data layer provides to the consumers. Granted that the current definition is tied to the REST API though but this is consequential to me.

Reply:
Pushed an update that tones down the WordPress REST API in favor of a more general approach.